### ISSUE TITLE
Submitting Ruby Prework Assignment

### ISSUE THREAD

**Issue by hvfenrir** [hvfenrir/First-Blog-Ruby#1]
Here is my completed prework assignment! Thank you
/cc @coderschoolreview

**Comment by coderschoolreview** [comment]
### Thank you for your Rails submission, hvfenrir!

:no_entry_sign: Unfortunately **there are some problems** with your submission:
- I couldn't find how much time you spent on the assignment. Please include the number of hours spent on your assignment in your `README.md`.
- I couldn't find a Heroku deploy link listed in your `README.md`. Please note the URL where you have deployed your app.

**Comment by hvfenrir** [comment]
Please check this link [README.md](https://github.com/hvfenrir/First-Blog-Ruby/blob/master/README.md) again. Thank you :)
/cc @coderschoolreview

**Comment by harley** [comment]
Hi @hvfenrir. thanks for updating the README file. Can you also include the user stories from the sample README file we provide in the prework?

Good job submitting the app! Currently your app is not good enough to be admitted, but if you have time feel free to improve it further and let us know we will review this submission again :)

Good luck
cc @coderschoolreview

**Comment by hvfenrir** [comment]
Sorry for my mistake. i'll finish that soon. ;)

**Comment by hvfenrir** [comment]
Please check this link [README.md](https://github.com/hvfenrir/First-Blog-Ruby/blob/master/README.md) again. Thank you :)
/cc @coderschoolreview

**Comment by hvfenrir** [comment]
Can you check my prework again. Thank you!
/cc @coderschoolreview

**Comment by hoksilato** [comment]
Hi @hvfenrir   

Thanks for your submitting. Good work :thumbsup:.

In summary, your submission is good. Great job with validations on article. Please fix below error:
- We didn't see any articles about the author.

Below are some **suggestions** that can improve your app:
- You should use Redcarpet to render on article detail page too.
- You should not allow user to delete the article about yourself.
- For better result, you should search with case insensitive.
- You can apply validations for comment too.

Let us know if you have any questions.

/cc @coderschoolreview @harley

**Comment by hvfenrir** [comment]
Thank for your feedback :D
I have a question is "We didn't see any articles about the author" you mean About page ??
/cc @hoksilato

**Comment by hoksilato** [comment]
Yes @hvfenrir, I looked for it on your Heroku demo. Actually, it's not necessary a about page, just an article is enough.

**Comment by hvfenrir** [comment]
Thank you :D
